odoo/addons/base/views/res_partner_views.xml

```diff
@@ -208,10 +208,16 @@
                         <group>
                             <field name="function" placeholder="e.g. Sales Director"
                                 attrs="{'invisible': [('is_company','=', True)]}"/>
-                            <field name="phone" widget="phone"/>
-                            <field name="mobile" widget="phone"/>
+                            <div id="base_phone">
```

Comment:
Hello, we usually don't use `id` inside templates because these have to be unique on a page to be HTML compliant, and it might be that several views for the same object (or more generally several widgets/components) are displayed at the same time on the page.

Reply:
Hello!
Thanks for the quick feedback :slightly_smiling_face: 
I understand your point indeed, but the use of an `id` makes it easier to target the `div` from other modules and place the necessary elements inside, instead of using a `position="replace"` which causes other kinds of problems :wink: 
Do you suggest I get rid of the `id` altogether or, that I come up with a more "elaborated" `id` to ensure it will be unique ?

Reply:
Just to answer in general, and as answered below by TDE, we usually just use a made up `name` attribute in that case.
Indeed you can target anything you want with xpath, it doesn't have to be an `id` specifically.